### PR TITLE
docs(*): Added api links in topics

### LINKED ISSUE #712: Add TypeDoc references in topics
Topics to update:
- [x] Avatar
- [x] Badge
- [x] Bottom Navigation
- [x] Button
- [x] Button Group
- [x] Checkbox 
- [x] Icon
- [x] Input and Label
- [x] Input Group
- [x] Mask
- [x] Radio Button/ Radio Group
- [x] Ripple
- [x] Switch 
- [x] Tabs
- [x] Time Picker
 
Grid features:
- [x] Column moving
- [x] Filtering
- [x] Column resizing
- [x] Conditional Cell Styling

### PR DESCRIPTION
**Pull request by sboykova**
Closes #712

### CODE REVIEW

File: en/components/grid_filtering.md [hunk 1]
```diff
@@ -29,16 +29,16 @@ There's a default filtering strategy provided out of the box, as well as all the
...
 
-You can filter any column or a combination of columns through the grid API. The grid exposes several methods for this task - `filter`, `filterGlobal ` and `clearFilter`.
+You can filter any column or a combination of columns through the grid API. The grid exposes several methods for this task - [`filter`](https://staging.infragistics.local/products/ignite-ui-angular/docs/typescript/classes/igxgridcomponent.html#filter), [`filterGlobal`](https://staging.infragistics.local/products/ignite-ui-angular/docs/typescript/classes/igxgridcomponent.html#filterglobal) and [`clearFilter`](https://staging.infragistics.local/products/ignite-ui-angular/docs/typescript/classes/igxgridcomponent.html#clearfilter).
 
-*   `filter` - filter a single column or a combination of columns.
+*   [`filter`]([`filter`](https://staging.infragistics.local/products/ignite-ui-angular/docs/typescript/classes/igxgridcomponent.html#filter)) - filter a single column or a combination of columns.
```
Comment (npaunov) [inline en/components/grid_filtering.md]: The link is not working, there is an extra ")" at the end.

File: en/components/time_picker.md [hunk 2]
```diff
@@ -155,9 +155,9 @@ And there we have it:
 
 We have seen how to make use of the  API (properties, events, methods) so that we configure the time picker per our requirements and interact with it programatically. Now we want to go further and customize its input group look.
 
-To do that we need to decorate the nested ng-template inside the time picker with IgxTimePickerTemplate directive. ng-template context exposes the following members: `openDialog` method can be used to open the time picker dialog; `displayValue` property contains the formatted value; `value` contains the real value. You could use those by declaring a variables in the ng-template element.
+To do that we need to decorate the nested ng-template inside the time picker with IgxTimePickerTemplate directive. ng-template context exposes the following members: [`openDialog`]({environment:angularApiUrl}/classes/igxtimepickercomponent.html#opendialog) method can be used to open the time picker dialog; `displayValue` property contains the formatted value; `value` contains the real value. You could use those by declaring a variables in the ng-template element.
```
Comment (npaunov) [inline en/components/time_picker.md]: **displayValue** should be modified to **displayTime**. Both **_displayTime_** and **_value_** should be links to the API documentation.

File: en/components/grid_filtering.md (hunk 1 above)
Comment (npaunov) [inline en/components/grid_filtering.md]: There is no API Reference for the Filtering Topic.

Comment (sboykova) [reply en/components/grid_filtering.md]: Fixed

File: en/components/time_picker.md (hunk 2 above)
Comment (sboykova) [reply en/components/time_picker.md]: Fixed

File: en/components/grid_filtering.md (hunk 1 above)
Comment (sboykova) [reply en/components/grid_filtering.md]: Added API References section

**Pull request merged**